frontend/src/metabase/components/ExpandingContent.jsx

```diff
@@ -59,6 +59,7 @@ class ExpandingContent extends Component {
           transition: `all ${duration}ms ease`,
           maxHeight: !animateHeight || isOpen ? maxHeight : 0,
           opacity: !animateOpacity || isOpen ? 1 : 0,
+          overflow: isOpen ? null : "hidden",
         }}
```

Comment:
```suggestion
        }}
        className={cx({"overflow-hidden": !isOpen})}
```

This probably has the same effect assuming there's no weird css precedence issues. This uses a class instead style, which is better in case we need to override something. It won't matter here though, so either way works for me.

Reply:
I like this approach a lot. It's even cleaner and I agree - we should always strive for classes over inline styles. And I learned something new about our system today, which is awesome. Thank you.

However, until we resolve the issue of including CSS twice, I'd leave this as a style for now.

Reply:
> until we resolve the issue of including CSS twice, I'd leave this as a style for now

We should fix that, but why avoid using classes before it's fixed?

Reply:
To avoid having it included twice. Although, one line will not make any difference - it just felt weird.
Ok, I'll switch to class.

Reply:
Oh I see. The "overflow-hidden" class is already in our styles, so I guess it's already being repeated haha. But I agree that even if you were adding a class here the additional line or two wouldn't make a difference.